sympy/physics/mechanics/_actuator.py

```diff
@@ -0,0 +1,243 @@
+"""Implementations of actuators for linked force and torque application.
+
+Notes
+=====
+
+This module is experimental and so is named with a leading underscore to
+indicate that the API is not yet stabilized and could be subject to breaking
+changes.
+
+"""
+
+from __future__ import annotations
+
+from abc import ABC, abstractmethod
+
+from sympy.core.backend import Basic
+from sympy.core.expr import Expr
+from sympy.physics.mechanics import Point, Vector
+from sympy.physics.mechanics._pathway import PathwayBase
+
+
+__all__ = ['ForceActuator']
+
+
+class ActuatorBase(ABC):
+    """Abstract base class for all actuator classes to inherit from.
+
+    Notes
+    =====
+
+    Instances of this class cannot be directly instantiated by users. However,
+    it can be used to created custom actuator types through subclassing.
+
+    """
+
+    def __init__(self) -> None:
+        """Initializer for ``ActuatorBase``."""
+        pass
+
+    @abstractmethod
+    def compute_loads(self, force: Expr) -> list[tuple[Point, Vector]]:
```

Comment:
I would not expect the `force` argument here. Besides that I would rename it to `to_loads`, for `PathWay` it was more sensible to use [`compute_loads`](https://github.com/sympy/sympy/pull/24914#discussion_r1137056068), due it taking `force` as an input argument

Reply:
I agree with your point here. I've renamed to `to_loads`. The extra `force` argument was a copy-paste typo, now removed, thanks for spotting. Addressed in https://github.com/sympy/sympy/pull/24966/commits/ae88012312c48ec2555edb66fcff76b815942f13.